Library/.rubocop.yml

```diff
@@ -186,6 +186,12 @@ Layout/LineLength:
                     ' "~/Library/Application Support/', '"~/Library/Caches/', '"~/Application Support',
                     ' was verified as official when first introduced to the cask']
 
+# Allow typographic quotes in comments.
+Style/AsciiComments:
+  AllowedChars:
+    - “
```

Comment:
Would like to avoid these.

Reply:
I think these are much nicer to look at when reading documentation.

Reply:
I disagree with having them inline (and the RuboCop defaults/existing style do too). Can our documentation generator not be configured to generate these instead?

Reply:
> Can our documentation generator not be configured to generate these instead?

I'm don't think so, but what advantage would that have over just using typographic quotes in the first place?

I mean you can use the HTML `<q>` tag or escape codes, but that just makes it harder to read (and write) inline, so I also see no advantage to using that.

Reply:
> I'm don't think so, but what advantage would that have over just using typographic quotes in the first place?

We haven't used them anywhere until this PR and this would render them correctly.

I don't find them inherently easier to read but making the documentation generator (YARD) or a preprocessor do this seems like a reasonable middle ground.

Reply:
Reworded these to not use quotes.

Reply:
Thanks!